### DISCUSSION TITLE
Request for Usearch

### DISCUSSION THREAD

**Discussion by utpalh** [Homebrew/discussions#3798] [Casks]
Usearch is available for free with 32 bit version. however the mac with M1 chip does not run. can anyone help to make it available . Link to the download site and link.

https://drive5.com/usearch/download.html

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Adding-Software-to-Homebrew

**Reply by utpalh** [reply]
thanks for the response. 
I really don't know much on git. 

could you help me 



cask "usearch" do
  version "11.0"
  sha256 :no_check

  url "https://drive5.com/downloads/usearch11.0.667_win32.exe.gz"
  name "usearch"
  desc "Extreme high-throughput sequence analysis. Orders of magnitude faster than BLAST."
  homepage "https://drive5.com/usearch/"

  app "usearch.app"
end

**Reply by p-linnane** [reply]
@utpalh Please open a pull request and I will help you work through it.
